appyters/Reactome_Harmonizome_ETL/requirements.txt

```diff
@@ -1,6 +1,14 @@
+anndata
 appyter @ git+https://github.com/MaayanLab/appyter
+bokeh
 harmonizome-helper @ git+https://github.com/MaayanLab/ETL-appyter-utilities
```

Comment:
Explicitly reference the new version of this package, namely:
`harmonizome-helper @ git+https://github.com/MaayanLab/ETL-appyter-utilities@v0.2.2`